jupyter-js-widgets/package.json

```diff
@@ -54,8 +54,7 @@
     "webpack": "^1.12.11"
   },
   "dependencies": {
-    "backbone": "1.2.0",
-    "bootstrap": "^3.3.5",
+    "backbone": "^1.2.3",
```

Comment:
Yes, we've had this breakage more than once. :(  It's unfortunate that everyone can't agree to semver.  I wonder what Backbone's version system is.  Time related?  Arbitrary?